src/panel/pets.ts

```diff
@@ -816,7 +816,7 @@ export class RubberDuck extends BasePetType {
     }
     hello(): string {
         // TODO: #190 Add a custom message for duck
-        return ` says hello 👋!`;
+        return ` Hi, I'm Donald Duck and I love to Quack around 👋!`;
```

Comment:
I'm gonna leave this as "I love to quack around", because it'll be confusing to have the name of the duck and then a message saying they're called Donald Duck

Reply:
Okay